docs/reference/snapshot-restore/apis/repo-analysis-api.asciidoc

```diff
@@ -39,9 +39,6 @@ POST /_snapshot/my_repository/_analyze?blob_count=10&max_blob_size=1mb&timeout=1
 <<privileges-list-cluster,cluster privilege>> to use this API. For more
 information, see <<security-privileges>>.
 
-* If the <<operator-privileges,{operator-feature}>> is enabled, only operator
-users can use this API.
-
```

Comment:
It needs to be deleted from this page as well
https://github.com/elastic/elasticsearch/blob/2d45e47bb5abf66ad80a06a7abf60de26ce5f147/docs/reference/security/operator-privileges/operator-only-functionality.asciidoc?plain=1#L24